setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 120
-ignore = E226,W291,W293,W503,F999,E305,F405,W504,N
+ignore = E226,W291,W293,W503,F999,E305,F405,W504,N,D,B
```

Comment:
Eventually we will want to include these. Personally I don't see why the `B` can't checked. That is meant to detect bugs (flake8-bugbear) and I haven't found any cases in my own work where it wasn't wrong about something.

Reply:
I'll remove the B and prep a separate pr to fix any issues it reports.